server_addon/maya/client/ayon_maya/plugins/publish/extract_maya_scene_raw.py

```diff
@@ -96,7 +96,7 @@ def process(self, instance):
                           "preserve_references"
                       ],
                       constructionHistory=True,
-                      shader=True,
+                      shader=True if instance.data.get("shader", True) else False,
```

Comment:
Could this just be like this?
```suggestion
                      shader=instance.data.get("shader", True),
```

Reply:
Agreed.